libraries/fwd/computeFwd/compute_fwd.cpp

```diff
@@ -1893,44 +1893,45 @@ ComputeFwd::~ComputeFwd()
-    int            ncomp    = 0;
-
-    FwdCoilSet*             megcoils = NULL;     /* The coil descriptions */
-    FwdCoilSet*             compcoils = NULL;    /* MEG compensation coils */
-    MneCTFCompDataSet*      comp_data  = NULL;
-    FwdCoilSet*             eegels = NULL;
-    FwdEegSphereModelSet*   eeg_models = NULL;
-
-    MneNamedMatrix* meg_forward      = NULL;    /* Result of the MEG forward calculation */
-    MneNamedMatrix* eeg_forward      = NULL;    /* Result of the EEG forward calculation */
-    MneNamedMatrix* meg_forward_grad = NULL;    /* Result of the MEG forward gradient calculation */
-    MneNamedMatrix* eeg_forward_grad = NULL;    /* Result of the EEG forward gradient calculation */
-    int            k;
-    fiffId         mri_id  = NULL;
-    fiffId         meas_id = NULL;
-    FILE           *out = NULL;     /* Output filtered points here */
-
-    FwdCoilSet*       templates = NULL;
-    FwdEegSphereModel* eeg_model = NULL;
-    FwdBemModel*       bem_model = NULL;
-
-    QString qPath;
-    QFile file;
+    // TODO: This only temporary until we have the fwd dlibrary refactored. This is only done in order to provide easy testing in test_forward_solution.
+//    bool                res = false;
+//    MneSourceSpaceOld*  *spaces = NULL;  /* The source spaces */
+//    int                 nspace  = 0;
+//    int                 nsource = 0;     /* Number of source space points */
+
+//    FiffCoordTransOld* mri_head_t = NULL;   /* MRI <-> head coordinate transformation */
+//    FiffCoordTransOld* meg_head_t = NULL;   /* MEG <-> head coordinate transformation */
+
+//    fiffChInfo     megchs   = NULL; /* The MEG channel information */
+//    int            nmeg     = 0;
+//    fiffChInfo     eegchs   = NULL; /* The EEG channel information */
+//    int            neeg     = 0;
+//    fiffChInfo     compchs = NULL;
+//    int            ncomp    = 0;
+
+//    FwdCoilSet*             megcoils = NULL;     /* The coil descriptions */
+//    FwdCoilSet*             compcoils = NULL;    /* MEG compensation coils */
+//    MneCTFCompDataSet*      comp_data  = NULL;
+//    FwdCoilSet*             eegels = NULL;
+//    FwdEegSphereModelSet*   eeg_models = NULL;
+
+//    MneNamedMatrix* meg_forward      = NULL;    /* Result of the MEG forward calculation */
+//    MneNamedMatrix* eeg_forward      = NULL;    /* Result of the EEG forward calculation */
+//    MneNamedMatrix* meg_forward_grad = NULL;    /* Result of the MEG forward gradient calculation */
+//    MneNamedMatrix* eeg_forward_grad = NULL;    /* Result of the EEG forward gradient calculation */
+//    int            k;
+//    fiffId         mri_id  = NULL;
+//    fiffId         meas_id = NULL;
+//    FILE           *out = NULL;     /* Output filtered points here */
+
+//    FwdCoilSet*       templates = NULL;
+//    FwdEegSphereModel* eeg_model = NULL;
+//    FwdBemModel*       bem_model = NULL;
+
+//    QString qPath;
+//    QFile file;
 
```

Comment:
What happened here?

Reply:
This only temporary until we have the fwd library refactored. This is only done in order to provide easy testing and access to the results in test_forward_solution. This is a workaround because the reading and writing is incompatible between our old and new (mne-c) MNEForwardSolution/MNESourceSpace classes.